packages/MarkdownEditor-Core.package/MarkdownInlineParser.class/instance/parse..st

```diff
@@ -0,0 +1,7 @@
+parsing
+parse: aString
+
+	self tokenize: aString.
```

Comment:
I think it is a bit weird to have the tokens as a instance variable (with the attributes, you could make more operations possible, but tokens are only used once). I'd suggest to make them a temp variable here.

Reply:
My thinking was that basically the whole inline Parser is one huge method object. Thus we want most of the variables to be instance so that we don't have to keep tossing them around as parameters. But in other parts of the class we haven't done this which makes this part here look weird. I would like the make it more into a method object and introduce even more instance variables which would make decomposing some of the methods much easier

Reply:
Passing them around as `aCollectionOfMarkdownTokens` would make the lines look like giant deformities. 😃 

@jonaskordt Which parts do you mean?

Reply:
for example we could split up the `interpretTokens` method in smaller methods if we created an instance variable for `currentToken` and maybe some more